test-cases/javascript/interactive/js-delete.html

```diff
@@ -1,8 +1,8 @@
 <!DOCTYPE html>
 <meta charset="utf-8">
-<title>CrawlMaze - Testbed for Web Crawlers - form tag</title>
+<title>CrawlMaze - Testbed for Web Crawlers</title>
```

Comment:
So the convention we use in the testbed is that we put some meaningful name here. For instance you could write: `CrawlMaze - Testbed for Web Crawlers - JS DELETE`